config/crd/bases/tower.ansible.com_ansibleexecutionenvironments.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: ansibleexecutionenvironment.tower.ansible.com
+spec:
+  group: tower.ansible.com
+  names:
+    kind: AnsibleExecutionEnvironment
+    listKind: AnsibleExecutionEnvironmentList
+    plural: AnsibleExecutionEnvironments
+    singular: AnsibleExecutionEnvironment
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: (Tech Preview) AnsibleExecutionEnvironment is the Schema for the ansibleexecutionenvironment API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              name:
+                type: string
+                description: The name for the executionenvironment
+              new_name:
+                type: string
+                description: The new name for the executionenvironment in case of a name change
+              credential:
+                type: string
+                description: The container registry credential for pulling the executionenvironment image
+              description:
+                type: string
+                description: The description for the executionenvironment
+              image:
+                type: string
+                description: The image for the executionenvironment
+              organization:
+                type: string
+                description: The organization for the executionenvironment
+              pull:
+                type: string
```

Comment:
Maybe the pull variable should be a enum, like in [awx.awx.execution_environment](https://docs.ansible.com/ansible/latest/collections/awx/awx/execution_environment_module.html#parameter-pull)

Reply:
How do you specify the choices when switching to enum?

Reply:
Got this fixed.